test/src/main/java/org/apache/accumulo/test/functional/GarbageCollectorIT.java

```diff
@@ -90,6 +90,9 @@ public void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite)
     cfg.setProperty(Property.GC_PORT, "0");
     cfg.setProperty(Property.TSERV_MAXMEM, "5K");
     cfg.setProperty(Property.TSERV_MAJC_DELAY, "1");
+    // reduce the batch size significantly in order to cause the integration tests to have
+    // to process many batches of deletion candidates.
+    cfg.setProperty(Property.GC_CANDIDATE_BATCH_SIZE, "256K");
```

Comment:
Do you know if this size caused multiple batches to be processed in the IT?  If not could make it smaller.

Reply:
@keith-turner, yes. That was the reasoning behind the modification. With the gcLotsOfCandidatesIT test it has to process a lot of batches.